relayer/middleware/wallet/wallet.middleware.ts

```diff
@@ -96,9 +111,60 @@ export interface WalletOpts {
     [k in ChainId]: any[];
   }>;
   logger?: Logger;
+  metrics?: {
+    registry: Registry;
+  };
 }
 
-export function wallets(opts: WalletOpts): Middleware<WalletContext> {
+const networks = {
+  [Environment.MAINNET]: {
+    [CHAIN_ID_ETH]: "mainnet",
+  },
+  [Environment.TESTNET]: {
```

Comment:
Yeah I think setting the name for each testnet was the right move for the monitor, having said that, it might be better to also support `testnet` and have that point to the name of the default testnet network for that chain. For instance, in celo it'll be alfajores